sklearn/linear_model/randomized_l1.py

```diff
@@ -58,6 +60,8 @@ def _resample_model(estimator_func, X, y, scaling=.5, n_resampling=200,
     return scores_
 
 
+@deprecated("The class BaseRandomizedLinearModel is deprecated in 0.19"
```

Comment:
Hm this should not have been public, but given that it was, deprecating it is the right thing to do.